swim/member.go

```diff
@@ -166,6 +187,23 @@ func (c Change) validateOutgoing() Change {
 	return c
 }
 
+func (c *Change) populateSubject(m *Member) {
```

Comment:
I have been thinking about doing it in a constructor but this gives a bit more control when a change needs to be constructed which information is being copied to a change. I will revisit it now that the all the call sites are known.